bedrock/settings/base.py

```diff
@@ -1046,6 +1047,22 @@ def get_app_name(hostname):
 CONTENTFUL_NOTIFICATION_QUEUE_SECRET_ACCESS_KEY = config("CONTENTFUL_NOTIFICATION_QUEUE_SECRET_ACCESS_KEY", default="", raise_error=False)
 CONTENTFUL_NOTIFICATION_QUEUE_WAIT_TIME = config("CONTENTFUL_NOTIFICATION_QUEUE_WAIT_TIME", default="10", parser=int, raise_error=False)
 
+CONTENTFUL_HOMEPAGE_LOOKUP = {
+    # TEMPORARY lookup table for which Contentful `connectHomepage` page ID to get for which locale
+    "en-US": "58YIvwDmzSDjtvpSqstDcL",
+    "de": "4k3CxqZGjxXOjR1I0dhyto",
+}
+
+CONTENTFUL_LOCALE_ACTIVATION_PERCENTAGE = config(
+    "CONTENTFUL_LOCALE_ACTIVATION_PERCENTAGE",
+    default="60",
```

Comment:
We should do the `if DEV` bit here so that demos or local dev can set this even in DEV mode. e.g.

```python
CONTENTFUL_LOCALE_ACTIVATION_PERCENTAGE = config(
    "CONTENTFUL_LOCALE_ACTIVATION_PERCENTAGE",
    default="1" if DEV else "60",
    parser=float,
)
```